src/libs/nav2_util/src/map/map_draw.c

```diff
@@ -25,6 +25,7 @@
  * CVS: $Id: map_draw.c 7057 2008-10-02 00:44:06Z gbiggs $
 **************************************************************************/
 
+#pragma GCC diagnostic ignored "-Wpedantic"
```

Comment:
Can we end the pragma after the offending include statements? I read that there is a 'push' and 'pop' for pragmas supported in some compilers to allow for this.

Reply:
[GCC diagnostics docs](https://gcc.gnu.org/onlinedocs/gcc/Diagnostic-Pragmas.html)
**Summary**
```c++
#pragma GCC diagnostic push
#pragma GCC diagnostic ignored "-Wpedantic"
<code>
#pragma GCC diagnostic pop
```

Reply:
Yep, that's what I was thinking. @mhpanah can you add this, but only around the offending include statements. That way we can still catch warnings in our own code.

Reply:
@mkhansen-intel Yes we can, but in this particular case the pragma is in the c file and it won't offend other files.

Reply:
@mkhansen-intel Added the pragma around tf2/util.h file.